src/coordination/include/coordination/utils.hpp

```diff
@@ -0,0 +1,101 @@
+// Copyright 2024 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+#pragma once
+
+#include <optional>
+#include <range/v3/view.hpp>
+#include <ranges>
+#include <string>
+#include <unordered_map>
+#include "dbms/constants.hpp"
+#include "replication_coordination_glue/common.hpp"
+
+namespace memgraph::coordination {
+
+inline std::string ChooseMostUpToDateInstance(
+    const std::vector<std::pair<std::string, replication_coordination_glue::DatabaseHistories>>
```

Comment:
using Instances or something else = std::vector<std::pair...

Reply:
you can do using some type for pair = std::pair,... since you are using it at multiple places

Reply:
make sense